### PR TITLE
x11-wm/i3: Stabilize v4.17.1 and remove older versions

### PR DESCRIPTION
**Pull request by ntnn**
Package-Manager: Portage-2.3.88, Repoman-2.3.20
Signed-off-by: Nelo-T. Wallus <nelo@wallus.de>

### CODE REVIEW

**Comment by a17r** [comment]
Please check your bugmail: https://bugs.gentoo.org/716952

File: x11-wm/i3/i3-4.17.1-r1.ebuild [hunk 1]
```diff
@@ -11,7 +11,7 @@ SRC_URI="https://i3wm.org/downloads/${P}.tar.bz2"
 
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="~amd64 ~arm64 ~x86"
+KEYWORDS="amd64 ~arm64 x86"
```
Comment (thesamesam) [inline x11-wm/i3/i3-4.17.1-r1.ebuild:14]: Please stabilise via Bugzilla.

File: x11-wm/i3/Manifest [hunk 2]
```diff
@@ -1,6 +1,2 @@
-DIST i3-4.14.tar.gz 3936748 BLAKE2B 42678130be56bc695e5700e978ddbd34bfdaeda7a65252a05568581d76fe613e80e710f3a507110e7988077fe62048bf984d4009e722d877b178206cef600221 SHA512 fe3db78813987a15ed93d182968bcd5139e1d03f29d3e8effd9fce59f87bc8309407af0b2fa5f1cd83b8583e50ea0ade6b8eafc5dda6dea9161832dc9cebfdfe
```
Comment (thesamesam) [inline x11-wm/i3/Manifest:1]: Do removal in a separate commit.
